CMakeLists.txt

```diff
@@ -24,9 +24,16 @@ FetchContent_MakeAvailable(bm_common)
 
 #TODO: Configuration will determine RTOS and IP stack here
 
+# TODO - is this needed here or leave it in the application cmake files?
+set(COMPILE_FLAGS
+    -DCBOR_CUSTOM_ALLOC_INCLUDE="tinycbor_alloc.h"
+    -DCBOR_PARSER_MAX_RECURSIONS=10
+)
+
```

Comment:
Personally I like this here, as long as we can make sure that the max_recursions are fairly compatible across platforms :)